android/build.gradle

```diff
@@ -12,8 +12,8 @@ android {
     buildToolsVersion _buildToolsVersion
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
```

Comment:
I am changing the Java version based on the notes in the Android SDK docs[here](https://tokbox.com/developer/sdks/android/#-creating-your-own-app-using-the-opentok-android-sdk).

Reply:
Yes just saw it